SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -1046,6 +1063,13 @@ void USpatialNetDriver::NotifyActorDestroyed(AActor* ThisActor, bool IsSeamlessT
 
 void USpatialNetDriver::Shutdown()
 {
+	UGameInstance* GameInstance = GetGameInstance();
```

Comment:
Can `GameInstance` actually be nullptr when using our `GetGameInstance()`? Should we ensure here?

Reply:
It is actually null. So by the point we reach this shutdown, the manager has already been cleaned up. I'd rather not have a hard requirement on this sequencing though, so I left the cleanup code just in case.